package.json

```diff
@@ -60,18 +60,23 @@
     "winston": "^2.4.0"
   },
   "devDependencies": {
+    "@types/jest": "^22.1.4",
```

Comment:
version of types should == version of jest

Reply:
(also for `supertest` and `superagent`)

Reply:
I've update the versions with the most similar one, but in case of supertest the package has version 3 and type has version 2.

Reply:
in this case, unfortunately, we're forced to use version 2 or you may incur in the risk of having typescript accept some code that in fact will not work (because the API from v2 to v3 has changed) -> typescript relies on the type file, not on the actual javascript file in the lib

Reply:
ok, I've downgraded the package